Tasks/google-play-release/GooglePlay.ts

```diff
@@ -442,35 +267,31 @@ async function addChangelog(edits: any, languageCode: string, changeLog: string,
  * @returns {string} change log file content as a string.
  */
 function getChangelog(changelogFile: string): string {
-    let changelog: string;
     tl.debug(`Reading change log from ${changelogFile}`);
-
     try {
-        changelog = fs.readFileSync(changelogFile).toString();
+        return fs.readFileSync(changelogFile).toString();
     } catch (e) {
         tl.debug(`Change log reading from ${changelogFile} failed`);
         tl.debug(e);
         throw new Error(tl.loc('CannotReadChangeLog', changelogFile));
     }
-
-    return changelog;
 }
 
 /**
- * Adds all changelogs found in directory to an edit. Pulls version code from file name. Failing this, assumes the global version code inferred from apk
+ * Adds all release notes found in directory to an edit. Pulls version code from file name. Failing this, assumes the global version code inferred from apk
  * Assumes authorized
  * @param {string} languageCode Language code (a BCP-47 language tag) of the localized listing to update
- * @param {string} directory Directory with a changesogs folder where changelogs can be found.
+ * @param {string} directory Directory with a changesogs folder where release notes can be found.
  * @returns nothing
  */
-async function addAllChangelogs(edits: any, apkVersionCodes: any, languageCode: string, directory: string) {
-    let changelogDir: string = path.join(directory, 'changelogs');
+async function addAllReleaseNotes(apkVersionCodes: any, languageCode: string, directory: string): Promise<googleutil.ReleaseNotes[]> {
+    const changelogDir: string = path.join(directory, 'changelogs');
 
-    let changelogs: string[] = fs.readdirSync(changelogDir).filter(subPath => {
+    const changelogs: string[] = fs.readdirSync(changelogDir).filter(subPath => {
```

Comment:
This block is mostly duplicated in `addMetadata`.  You could pull out a helper function:
```ts
function filterDirectoryContents(directory: string, filter: (stats: Stats) => boolean): string[] {
    return fs.readdirSync(directory).filter(subPath => {
        try {
            const fullPath: string = path.join(directory, subPath);
            tl.debug(`Checking path ${fullPath}`);
            return filter(tl.stats(fullPath));
        } catch (e) {
            tl.debug(`Failed to stat path ${subPath}:`);
            tl.debug(e);
            tl.debug('Ignoring...');
            return false;
        }
    });
}

// ...
const changelogs: string[] = filterDirectoryContents(changelogDir, stat => stat.isFile());
// ...
const metadataLanguageCodes: string[] = filterDirectoryContents(metadataRootDirectory, stats.isDirectory());
```
It jumped out at me just because the logging was different between the two, so it'd be nice to keep that similar.